tests.foreman.cli.test_host.HostCreateTestCase.test_negative_create_with_content_source: Content source with given ID not found

```
Error Message

robottelo.cli.factory.CLIFactoryError: Failed to create Host with data: {   "architecture": null,   "architecture-id": "1",   "ask-root-password": null,   "autoheal": null,   "build": null,   "comment": null,   "compute-attributes": null,   "compute-profile": null,   "compute-profile-id": null,   "compute-resource": null,   "compute-resource-id": null,   "content-source-id": 31792,   "content-view": null,   "content-view-id": "373",   "domain": null,   "domain-id": "87",   "enabled": null,   "environment": null,   "environment-id": null,   "hostgroup": null,   "hostgroup-id": null,   "hostgroup-title": null,   "hypervisor-guest-uuids": null,   "image": null,   "image-id": null,   "interface": null,   "ip": "140.47.125.252",   "kickstart-repository-id": null,   "lifecycle-environment": null,   "lifecycle-environment-id": "340",   "location": null,   "location-id": "2",   "mac": "9c:d6:b2:d2:80:a4",   "managed": null,   "medium": null,   "medium-id": "109",   "model": null,   "model-id": null,   "name": "srBKllKcOY",   "operatingsystem": null,   "operatingsystem-id": "1",   "organization": "y9R5CI",   "organization-id": null,   "overwrite": null,   "owner": null,   "owner-id": null,   "owner-type": null,   "parameters": null,   "partition-table": null,   "partition-table-id": "94",   "progress-report-id": null,   "provision-method": null,   "puppet-ca-proxy": null,   "puppet-ca-proxy-id": null,   "puppet-class-ids": null,   "puppet-classes": null,   "puppet-proxy": null,   "puppet-proxy-id": null,   "pxe-loader": null,   "realm": null,   "realm-id": null,   "root-password": "iqVclsjv",   "service-level": null,   "subnet": null,   "subnet-id": null,   "volume": null } Command "host create" finished with return_code 65 stderr contains following message: [ERROR 2018-06-23T05:53:54 API] 422 Unprocessable Entity [ERROR 2018-06-23T05:53:54 Exception] Content source with given ID not found Could not create the host:   Content source with given ID not found

Stacktrace

self = <tests.foreman.cli.test_host.HostCreateTestCase testMethod=test_negative_create_with_content_source>

    @tier1
    def test_negative_create_with_content_source(self):
        """Attempt to create a host with invalid content source specified
    
            :id: d92d6aff-4ad3-467c-88a8-5a5e56614f58
    
            :BZ: 1260697
    
            :expectedresults: Host was not created
    
            :CaseImportance: Medium
            """
        with self.assertRaises(CLIBaseError):
            make_fake_host({
                'content-source-id': gen_integer(10000, 99999),
                'content-view-id': self.DEFAULT_CV['id'],
                'lifecycle-environment-id': self.LIBRARY['id'],
>               'organization': self.new_org['name'],
            })

tests/foreman/cli/test_host.py:388: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
robottelo/decorators/__init__.py:190: in cacheable_function
    new_object = func(options)
robottelo/cli/factory.py:1627: in make_fake_host
    return make_host(options)
robottelo/decorators/__init__.py:190: in cacheable_function
    new_object = func(options)
robottelo/cli/factory.py:1547: in make_host
    return create_object(Host, args, options)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def create_object(cli_object, options, values):
        """
        Creates <object> with dictionary of arguments.
    
        :param cli_object: A valid CLI object.
        :param dict options: The default options accepted by the cli_object
            create
        :param dict values: Custom values to override default ones.
        :raise robottelo.cli.factory.CLIFactoryError: Raise an exception if object
            cannot be created.
        :rtype: dict
        :return: A dictionary representing the newly created resource.
    
        """
        if values:
            diff = set(values.keys()).difference(set(options.keys()))
            if diff:
                logger.debug(
                    "Option(s) {0} not supported by CLI factory. Please check for "
                    "a typo or update default options".format(diff)
                )
        update_dictionary(options, values)
        try:
            result = cli_object.create(options)
        except CLIReturnCodeError as err:
            # If the object is not created, raise exception, stop the show.
            raise CLIFactoryError(
                u'Failed to create {0} with data:\n{1}\n{2}'.format(
                    cli_object.__name__,
                    json.dumps(options, indent=2, sort_keys=True),
>                   err.msg,
                )
            )
E           robottelo.cli.factory.CLIFactoryError: Failed to create Host with data:
E           {
E             "architecture": null,
E             "architecture-id": "1",
E             "ask-root-password": null,
E             "autoheal": null,
E             "build": null,
E             "comment": null,
E             "compute-attributes": null,
E             "compute-profile": null,
E             "compute-profile-id": null,
E             "compute-resource": null,
E             "compute-resource-id": null,
E             "content-source-id": 31792,
E             "content-view": null,
E             "content-view-id": "373",
E             "domain": null,
E             "domain-id": "87",
E             "enabled": null,
E             "environment": null,
E             "environment-id": null,
E             "hostgroup": null,
E             "hostgroup-id": null,
E             "hostgroup-title": null,
E             "hypervisor-guest-uuids": null,
E             "image": null,
E             "image-id": null,
E             "interface": null,
E             "ip": "140.47.125.252",
E             "kickstart-repository-id": null,
E             "lifecycle-environment": null,
E             "lifecycle-environment-id": "340",
E             "location": null,
E             "location-id": "2",
E             "mac": "9c:d6:b2:d2:80:a4",
E             "managed": null,
E             "medium": null,
E             "medium-id": "109",
E             "model": null,
E             "model-id": null,
E             "name": "srBKllKcOY",
E             "operatingsystem": null,
E             "operatingsystem-id": "1",
E             "organization": "y9R5CI",
E             "organization-id": null,
E             "overwrite": null,
E             "owner": null,
E             "owner-id": null,
E             "owner-type": null,
E             "parameters": null,
E             "partition-table": null,
E             "partition-table-id": "94",
E             "progress-report-id": null,
E             "provision-method": null,
E             "puppet-ca-proxy": null,
E             "puppet-ca-proxy-id": null,
E             "puppet-class-ids": null,
E             "puppet-classes": null,
E             "puppet-proxy": null,
E             "puppet-proxy-id": null,
E             "pxe-loader": null,
E             "realm": null,
E             "realm-id": null,
E             "root-password": "iqVclsjv",
E             "service-level": null,
E             "subnet": null,
E             "subnet-id": null,
E             "volume": null
E           }
E           Command "host create" finished with return_code 65
E           stderr contains following message:
E           [ERROR 2018-06-23T05:53:54 API] 422 Unprocessable Entity
E           [ERROR 2018-06-23T05:53:54 Exception] Content source with given ID not found
E           Could not create the host:
E             Content source with given ID not found

robottelo/cli/factory.py:140: CLIFactoryError

Standard Output

2018-06-23 05:51:46 - robottelo - INFO - Started setUpClass: tests.foreman.cli.test_host/HostCreateTestCase
2018-06-23 05:51:46 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62147710
2018-06-23 05:51:46 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:51:46 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv organization create --name="y9R5CI"'
2018-06-23 05:51:53 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Organization created,446,y9R5CI

2018-06-23 05:51:53 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62147710
2018-06-23 05:51:54 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62147b38
2018-06-23 05:51:54 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:51:54 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  organization info --id="446"'
2018-06-23 05:51:57 - robottelo.ssh - INFO - <<< stdout
Id:                     446
Title:                  y9R5CI
Name:                   y9R5CI
Description:            
Users:                  

Smart proxies:          
    <fqdn>
Subnets:                

Compute resources:      

Installation media:     

Templates:              
    Alterator default (provision)
    Alterator default finish (finish)
    Alterator default PXELinux (PXELinux)
    alterator_pkglist ()
    ansible_provisioning_callback ()
    ansible_tower_callback_script ()
    ansible_tower_callback_service ()
    Atomic Kickstart default (provision)
    AutoYaST default (provision)
    AutoYaST default iPXE (iPXE)
    AutoYaST default PXELinux (PXELinux)
    AutoYaST default user data (user_data)
    AutoYaST SLES default (provision)
    bmc_nic_setup ()
    Boot disk iPXE - generic host (Bootdisk)
    Boot disk iPXE - host (Bootdisk)
    chef_client ()
    coreos_cloudconfig ()
    CoreOS provision (provision)
    CoreOS PXELinux (PXELinux)
    create_users ()
    Discovery Debian kexec (kexec)
    Discovery Red Hat kexec (kexec)
    epel ()
    fix_hosts ()
    FreeBSD (mfsBSD) finish (finish)
    FreeBSD (mfsBSD) provision (provision)
    FreeBSD (mfsBSD) PXELinux (PXELinux)
    freeipa_register ()
    Grubby default (script)
    http_proxy ()
    idm_register ()
    Jumpstart default (provision)
    Jumpstart default finish (finish)
    Jumpstart default PXEGrub (PXEGrub)
    Junos default finish (finish)
    Junos default SLAX (provision)
    Junos default ZTP config (ZTP)
    Kickstart default finish (finish)
    Kickstart default iPXE (iPXE)
    Kickstart default PXEGrub (PXEGrub)
    Kickstart default PXEGrub2 (PXEGrub2)
    Kickstart default PXELinux (PXELinux)
    Kickstart default user data (user_data)
    kickstart_ifcfg_bonded_interface ()
    kickstart_ifcfg_bond_interface ()
    kickstart_ifcfg_generic_interface ()
    kickstart_ifcfg_get_identifier_names ()
    kickstart_networking_setup ()
    Kickstart oVirt-RHVH (provision)
    Kickstart oVirt-RHVH PXELinux (PXELinux)
    NX-OS default POAP setup (POAP)
    Preseed default (provision)
    Preseed default finish (finish)
    Preseed default iPXE (iPXE)
    Preseed default PXEGrub2 (PXEGrub2)
    Preseed default PXELinux (PXELinux)
    Preseed default user data (user_data)
    preseed_networking_setup ()
    puppet.conf ()
    puppetlabs_repo ()
    puppet_setup ()
    pxegrub2_chainload ()
    PXEGrub2 default local boot (PXEGrub2)
    pxegrub2_discovery ()
    PXEGrub2 global default (PXEGrub2)
    pxegrub_chainload ()
    PXEGrub default local boot (PXEGrub)
    pxegrub_discovery ()
    PXEGrub global default (PXEGrub)
    PXELinux chain iPXE (PXELinux)
    PXELinux chain iPXE UNDI (PXELinux)
    pxelinux_chainload ()
    PXELinux default local boot (PXELinux)
    PXELinux default memdisk (PXELinux)
    pxelinux_discovery ()
    PXELinux global default (PXELinux)
    rancheros_cloudconfig ()
    RancherOS provision (provision)
    RancherOS PXELinux (PXELinux)
    redhat_register ()
    remote_execution_ssh_keys ()
    saltstack_minion ()
    saltstack_setup ()
    Satellite Atomic Kickstart Default (provision)
    Satellite Kickstart Default (provision)
    Satellite Kickstart Default Finish (finish)
    Satellite Kickstart Default User Data (user_data)
    UserData default (user_data)
    WAIK default PXELinux (PXELinux)
    XenServer default answerfile (provision)
    XenServer default finish (finish)
    XenServer default PXELinux (PXELinux)
Domains:                

Environments:           

Hostgroups:             

Parameters:             

Created at:             2018/06/23 09:51:48
Updated at:             2018/06/23 09:51:48
Label:                  y9R5CI
Description:            
Red Hat Repository URL: https://cdn.redhat.com
Service Levels:


2018-06-23 05:51:57 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62147b38
2018-06-23 05:51:57 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b664cacc0
2018-06-23 05:51:57 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:51:57 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv lifecycle-environment create --name="KNkuZz" --organization-id="446" --prior="Library"'
2018-06-23 05:52:00 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Environment created,341,KNkuZz

2018-06-23 05:52:00 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b664cacc0
2018-06-23 05:52:00 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b664cacc0
2018-06-23 05:52:00 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:00 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  lifecycle-environment info --id="341" --organization-id="446"'
2018-06-23 05:52:03 - robottelo.ssh - INFO - <<< stdout
ID:                          341
Name:                        KNkuZz
Label:                       KNkuZz
Description:                 
Organization:                y9R5CI
Library:                     false
Prior Lifecycle Environment: Library


2018-06-23 05:52:03 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b664cacc0
2018-06-23 05:52:03 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b664caf98
2018-06-23 05:52:03 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:03 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  lifecycle-environment info --organization-id="446" --name="Library"'
2018-06-23 05:52:05 - robottelo.ssh - INFO - <<< stdout
ID:                          340
Name:                        Library
Label:                       Library
Description:                 
Organization:                y9R5CI
Library:                     true
Prior Lifecycle Environment:


2018-06-23 05:52:05 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b664caf98
2018-06-23 05:52:05 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b621ce320
2018-06-23 05:52:05 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:05 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view info --organization-id="446" --name="Default Organization View"'
2018-06-23 05:52:07 - robottelo.ssh - INFO - <<< stdout
ID:                     373
Name:                   Default Organization View
Label:                  167ef022-c314-4d0f-a27f-0f36b3742537
Composite:              false
Description:            
Content Host Count:     0
Organization:           y9R5CI
Yum Repositories:       

Docker Repositories:    

OSTree Repositories:    

Puppet Modules:         

Lifecycle Environments: 
 1) ID:   340
    Name: Library
Versions:               
 1) ID:        345
    Version:   1.0
    Published: 2018/06/23 09:51:49
Components:             

Activation Keys:


2018-06-23 05:52:07 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b621ce320
2018-06-23 05:52:07 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b626230b8
2018-06-23 05:52:07 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:07 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv content-view create --name="DiYlAgHqNC" --organization-id="446"'
2018-06-23 05:52:10 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Content view created,376,DiYlAgHqNC

2018-06-23 05:52:10 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b626230b8
2018-06-23 05:52:10 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b626230b8
2018-06-23 05:52:10 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:10 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view info --id="376"'
2018-06-23 05:52:12 - robottelo.ssh - INFO - <<< stdout
ID:                     376
Name:                   DiYlAgHqNC
Label:                  DiYlAgHqNC
Composite:              false
Description:            
Content Host Count:     0
Organization:           y9R5CI
Yum Repositories:       

Docker Repositories:    

OSTree Repositories:    

Puppet Modules:         

Lifecycle Environments: 

Versions:               

Components:             

Activation Keys:


2018-06-23 05:52:12 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b626230b8
2018-06-23 05:52:12 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b664f5cc0
2018-06-23 05:52:12 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:12 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view publish --id="376"'
2018-06-23 05:52:17 - robottelo.ssh - INFO - <<< stdout
{"content_view_id"=>376,
 "content_view_version_id"=>348,
 "composite_version_auto_published"=>[],
 "composite_view_publish_failed"=>[],
 "composite_auto_publish_task_id"=>[]}

2018-06-23 05:52:17 - robottelo.ssh - INFO - <<< stderr
Task b233427a-ce3d-41e7-bcd0-0a2253553b26 running: 0.0/1, 0%, elapsed: 00:00:00
Task b233427a-ce3d-41e7-bcd0-0a2253553b26 success: 1.0/1, 100%, 0.5/s, elapsed: 00:00:02
Task b233427a-ce3d-41e7-bcd0-0a2253553b26 success: 1.0/1, 100%, 0.5/s, elapsed: 00:00:02

2018-06-23 05:52:17 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b664f5cc0
2018-06-23 05:52:17 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62147f98
2018-06-23 05:52:17 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:17 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv content-view version list --content-view-id="376"'
2018-06-23 05:52:21 - robottelo.ssh - INFO - <<< stdout
ID,Name,Version,Lifecycle Environments
348,DiYlAgHqNC 1.0,1.0,Library

2018-06-23 05:52:21 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62147f98
2018-06-23 05:52:21 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b621ce320
2018-06-23 05:52:21 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:21 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view version promote --id="348" --to-lifecycle-environment-id="341" --organization-id="446"'
2018-06-23 05:52:26 - robottelo.ssh - INFO - <<< stderr
Task bdd0ec8c-79a0-4476-a21c-97c9e421456d running: 0.0/1, 0%, elapsed: 00:00:00
Task bdd0ec8c-79a0-4476-a21c-97c9e421456d success: 1.0/1, 100%, 0.5/s, elapsed: 00:00:02
Task bdd0ec8c-79a0-4476-a21c-97c9e421456d success: 1.0/1, 100%, 0.5/s, elapsed: 00:00:02

2018-06-23 05:52:26 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b621ce320
2018-06-23 05:52:26 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6261a908
2018-06-23 05:52:26 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:26 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv product create --description="YaRaqzmEdlvLVrvGdLVC" --label="ybPfGSBrqYXXXsTpdTRD" --name="NbncTlzKgRijikbRkOjZ" --organization-id="446"'
2018-06-23 05:52:32 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Product created,844,NbncTlzKgRijikbRkOjZ

2018-06-23 05:52:32 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6261a908
2018-06-23 05:52:32 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6261a908
2018-06-23 05:52:32 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:32 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  product info --id="844" --organization-id="446"'
2018-06-23 05:52:34 - robottelo.ssh - INFO - <<< stdout
ID:           844
Name:         NbncTlzKgRijikbRkOjZ
Label:        ybPfGSBrqYXXXsTpdTRD
Description:  YaRaqzmEdlvLVrvGdLVC
Sync State:   
Sync Plan ID: 
GPG:          
    GPG Key ID: 
    GPG Key:
Organization: y9R5CI
Readonly:     
Deletable:    
Content:


2018-06-23 05:52:34 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6261a908
2018-06-23 05:52:34 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62623780
2018-06-23 05:52:34 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:34 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv repository create --content-type="puppet" --name="LySUYKoEQHTZFtr" --product-id="844" --publish-via-http="true" --url="http://omaciel.fedorapeople.org/bagoftricks"'
2018-06-23 05:52:39 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Repository created,139,LySUYKoEQHTZFtr

2018-06-23 05:52:39 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62623780
2018-06-23 05:52:39 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62153c50
2018-06-23 05:52:39 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:39 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  repository info --id="139"'
2018-06-23 05:52:43 - robottelo.ssh - INFO - <<< stdout
ID:                 139
Name:               LySUYKoEQHTZFtr
Label:              LySUYKoEQHTZFtr
Organization:       y9R5CI
Red Hat Repository: no
Content Type:       puppet
Mirror on Sync:     yes
URL:                http://omaciel.fedorapeople.org/bagoftricks
Publish Via HTTP:   yes
Published At:       http://<fqdn>/pulp/puppet/6fd50ed3-5a4d-4d09-8665-d432a1c289d8/
Relative Path:      y9R5CI/Library/custom/ybPfGSBrqYXXXsTpdTRD/LySUYKoEQHTZFtr
Product:            
    ID:   844
    Name: NbncTlzKgRijikbRkOjZ
GPG Key:            

Sync:               
    Status: Not Synced
Created:            2018/06/23 09:52:36
Updated:            2018/06/23 09:52:37
Content Counts:     
    Puppet Modules: 0


2018-06-23 05:52:43 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62153c50
2018-06-23 05:52:43 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6262ed30
2018-06-23 05:52:43 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:43 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv repository synchronize --id="139"'
2018-06-23 05:52:54 - robottelo.ssh - INFO - <<< stderr
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab running: 0.0/1, 0%, elapsed: 00:00:00
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab running: 0.05/1, 5%, 0.0/s, elapsed: 00:00:02, ETA: 00:00:40
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab running: 0.05/1, 5%, 0.0/s, elapsed: 00:00:04, ETA: 00:01:20
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab running: 0.7/1, 70%, 0.1/s, elapsed: 00:00:06, ETA: 00:00:02
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab success: 1.0/1, 100%, 0.1/s, elapsed: 00:00:08
Task b8fb1a7e-03ce-4838-ab43-847038abd1ab success: 1.0/1, 100%, 0.1/s, elapsed: 00:00:08

2018-06-23 05:52:54 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6262ed30
2018-06-23 05:52:54 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6261abe0
2018-06-23 05:52:54 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:54 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv content-view create --name="ESxdAFJUfH" --organization-id="446"'
2018-06-23 05:52:56 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Content view created,386,ESxdAFJUfH

2018-06-23 05:52:56 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6261abe0
2018-06-23 05:52:56 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62153c50
2018-06-23 05:52:56 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:56 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view info --id="386"'
2018-06-23 05:52:59 - robottelo.ssh - INFO - <<< stdout
ID:                     386
Name:                   ESxdAFJUfH
Label:                  ESxdAFJUfH
Composite:              false
Description:            
Content Host Count:     0
Organization:           y9R5CI
Yum Repositories:       

Docker Repositories:    

OSTree Repositories:    

Puppet Modules:         

Lifecycle Environments: 

Versions:               

Components:             

Activation Keys:


2018-06-23 05:52:59 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62153c50
2018-06-23 05:52:59 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6202ce48
2018-06-23 05:52:59 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:52:59 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv content-view puppet-module add --author="robottelo" --name="generic_1" --content-view-id="386"'
2018-06-23 05:53:01 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Puppet module added to content view,20,generic_1

2018-06-23 05:53:01 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6202ce48
2018-06-23 05:53:01 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62636128
2018-06-23 05:53:01 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:01 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view publish --id="386"'
2018-06-23 05:53:20 - robottelo.ssh - INFO - <<< stdout
{"content_view_id"=>386,
 "content_view_version_id"=>355,
 "composite_version_auto_published"=>[],
 "composite_view_publish_failed"=>[],
 "composite_auto_publish_task_id"=>[]}

2018-06-23 05:53:20 - robottelo.ssh - INFO - <<< stderr
Task da9c110a-db9b-408a-9056-f25b4678228c planned: 0.0/1, 0%, elapsed: 00:00:00
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.16666666666666666/1, 16%, 0.1/s, elapsed: 00:00:02, ETA: 00:00:10
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.4/1, 40%, 0.1/s, elapsed: 00:00:04, ETA: 00:00:06
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.6666666666666666/1, 66%, 0.1/s, elapsed: 00:00:06, ETA: 00:00:03
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.8/1, 80%, 0.1/s, elapsed: 00:00:08, ETA: 00:00:02
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.8/1, 80%, 0.1/s, elapsed: 00:00:10, ETA: 00:00:02
Task da9c110a-db9b-408a-9056-f25b4678228c running: 0.8/1, 80%, 0.1/s, elapsed: 00:00:12, ETA: 00:00:03
Task da9c110a-db9b-408a-9056-f25b4678228c success: 1.0/1, 100%, 0.1/s, elapsed: 00:00:14
Task da9c110a-db9b-408a-9056-f25b4678228c success: 1.0/1, 100%, 0.1/s, elapsed: 00:00:14

2018-06-23 05:53:20 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62636128
2018-06-23 05:53:21 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6261af98
2018-06-23 05:53:21 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:21 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  content-view info --id="386"'
2018-06-23 05:53:23 - robottelo.ssh - INFO - <<< stdout
ID:                     386
Name:                   ESxdAFJUfH
Label:                  ESxdAFJUfH
Composite:              false
Description:            
Content Host Count:     0
Organization:           y9R5CI
Yum Repositories:       

Docker Repositories:    

OSTree Repositories:    

Puppet Modules:         
 1) ID:     20
    Name:   generic_1
    Author: robottelo
Lifecycle Environments: 
 1) ID:   340
    Name: Library
Versions:               
 1) ID:        355
    Version:   1.0
    Published: 2018/06/23 09:53:04
Components:             

Activation Keys:


2018-06-23 05:53:23 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6261af98
2018-06-23 05:53:23 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6202c048
2018-06-23 05:53:23 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:23 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv environment list --search="content_view="ESxdAFJUfH"" --per-page="10000"'
2018-06-23 05:53:25 - robottelo.ssh - INFO - <<< stdout
Id,Name
111,KT_y9R5CI_Library_ESxdAFJUfH_386

2018-06-23 05:53:25 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6202c048
2018-06-23 05:53:25 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62603908
2018-06-23 05:53:25 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:25 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  puppet-class info --name="generic_1" --environment="KT_y9R5CI_Library_ESxdAFJUfH_386"'
2018-06-23 05:53:29 - robottelo.ssh - INFO - <<< stdout
Id:                     33
Name:                   generic_1
Smart variables:        

Smart class parameters: 
    generic_1_scp_001
    generic_1_scp_002
    generic_1_scp_003
    generic_1_scp_004
    generic_1_scp_005
    generic_1_scp_006
    generic_1_scp_007
    generic_1_scp_008
    generic_1_scp_009
    generic_1_scp_010
    generic_1_scp_011
    generic_1_scp_012
    generic_1_scp_013
    generic_1_scp_014
    generic_1_scp_015
    generic_1_scp_016
    generic_1_scp_017
    generic_1_scp_018
    generic_1_scp_019
    generic_1_scp_020
    generic_1_scp_021
    generic_1_scp_022
    generic_1_scp_023
    generic_1_scp_024
    generic_1_scp_025
    generic_1_scp_026
    generic_1_scp_027
    generic_1_scp_028
    generic_1_scp_029
    generic_1_scp_030
    generic_1_scp_031
    generic_1_scp_032
    generic_1_scp_033
    generic_1_scp_034
    generic_1_scp_035
    generic_1_scp_036
    generic_1_scp_037
    generic_1_scp_038
    generic_1_scp_039
    generic_1_scp_040
    generic_1_scp_041
    generic_1_scp_042
    generic_1_scp_043
    generic_1_scp_044
    generic_1_scp_045
    generic_1_scp_046
    generic_1_scp_047
    generic_1_scp_048
    generic_1_scp_049
    generic_1_scp_050
    generic_1_scp_051
    generic_1_scp_052
    generic_1_scp_053
    generic_1_scp_054
    generic_1_scp_055
    generic_1_scp_056
    generic_1_scp_057
    generic_1_scp_058
    generic_1_scp_059
    generic_1_scp_060
    generic_1_scp_061
    generic_1_scp_062
    generic_1_scp_063
    generic_1_scp_064
    generic_1_scp_065
    generic_1_scp_066
    generic_1_scp_067
    generic_1_scp_068
    generic_1_scp_069
    generic_1_scp_070
    generic_1_scp_071
    generic_1_scp_072
    generic_1_scp_073
    generic_1_scp_074
    generic_1_scp_075
    generic_1_scp_076
    generic_1_scp_077
    generic_1_scp_078
    generic_1_scp_079
    generic_1_scp_080
    generic_1_scp_081
    generic_1_scp_082
    generic_1_scp_083
    generic_1_scp_084
    generic_1_scp_085
    generic_1_scp_086
    generic_1_scp_087
    generic_1_scp_088
    generic_1_scp_089
    generic_1_scp_090
    generic_1_scp_091
    generic_1_scp_092
    generic_1_scp_093
    generic_1_scp_094
    generic_1_scp_095
    generic_1_scp_096
    generic_1_scp_097
    generic_1_scp_098
    generic_1_scp_099
    generic_1_scp_100
    generic_1_scp_101
    generic_1_scp_102
    generic_1_scp_103
    generic_1_scp_104
    generic_1_scp_105
    generic_1_scp_106
    generic_1_scp_107
    generic_1_scp_108
    generic_1_scp_109
    generic_1_scp_110
    generic_1_scp_111
    generic_1_scp_112
    generic_1_scp_113
    generic_1_scp_114
    generic_1_scp_115
    generic_1_scp_116
    generic_1_scp_117
    generic_1_scp_118
    generic_1_scp_119
    generic_1_scp_120
    generic_1_scp_121
    generic_1_scp_122
    generic_1_scp_123
    generic_1_scp_124
    generic_1_scp_125
    generic_1_scp_126
    generic_1_scp_127
    generic_1_scp_128
    generic_1_scp_129
    generic_1_scp_130
    generic_1_scp_131
    generic_1_scp_132
    generic_1_scp_133
    generic_1_scp_134
    generic_1_scp_135
    generic_1_scp_136
    generic_1_scp_137
    generic_1_scp_138
    generic_1_scp_139
    generic_1_scp_140
    generic_1_scp_141
    generic_1_scp_142
    generic_1_scp_143
    generic_1_scp_144
    generic_1_scp_145
    generic_1_scp_146
    generic_1_scp_147
    generic_1_scp_148
    generic_1_scp_149
    generic_1_scp_150
    generic_1_scp_151
    generic_1_scp_152
    generic_1_scp_153
    generic_1_scp_154
    generic_1_scp_155
    generic_1_scp_156
    generic_1_scp_157
    generic_1_scp_158
    generic_1_scp_159
    generic_1_scp_160
    generic_1_scp_161
    generic_1_scp_162
    generic_1_scp_163
    generic_1_scp_164
    generic_1_scp_165
    generic_1_scp_166
    generic_1_scp_167
    generic_1_scp_168
    generic_1_scp_169
    generic_1_scp_170
    generic_1_scp_171
    generic_1_scp_172
    generic_1_scp_173
    generic_1_scp_174
    generic_1_scp_175
    generic_1_scp_176
    generic_1_scp_177
    generic_1_scp_178
    generic_1_scp_179
    generic_1_scp_180
    generic_1_scp_181
    generic_1_scp_182
    generic_1_scp_183
    generic_1_scp_184
    generic_1_scp_185
    generic_1_scp_186
    generic_1_scp_187
    generic_1_scp_188
    generic_1_scp_189
    generic_1_scp_190
    generic_1_scp_191
    generic_1_scp_192
    generic_1_scp_193
    generic_1_scp_194
    generic_1_scp_195
    generic_1_scp_196
    generic_1_scp_197
    generic_1_scp_198
    generic_1_scp_199
    generic_1_scp_200
Hostgroups:             

Environments:           
    KT_6LZOQo_Library_jEGrOAJjgV_25
    KT_BtVvtEwkYrbM_Library_klSDVDE_252
    KT_gARliDj_Library_PbHFji_140
    KT_LrSnKoIeJLT_Library_SqXVHHcCMg_163
    KT_NpnlzPXNJaU_Library_GbXsaOy_100
    KT_psuJTS_Library_XwGsPKxCcu_334
    KT_rzmRc2_Library_PxxzHiAyHT_23
    KT_VvWWLWO_Library_gRZgONet_196
    KT_WdNIks_Library_mprTXwLCstd_117
    KT_y9R5CI_Library_ESxdAFJUfH_386
    KT_ZhyiwbFJj_Library_amdmkF_127
    KT_ZMsvENJ_Library_oFqBEC_225
Parameters:


2018-06-23 05:53:29 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62603908
2018-06-23 05:53:29 - robottelo - INFO - Entities cleaner disabled
2018-06-23 05:53:29 - robottelo - DEBUG - Started Test: HostCreateTestCase/test_negative_create_with_content_source
2018-06-23 05:53:29 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62147e80
2018-06-23 05:53:29 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:29 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv proxy list --search="url = https://<fqdn>:9090" --per-page="10000"'
2018-06-23 05:53:31 - robottelo.ssh - INFO - <<< stdout
Id,Name,URL,Features
1,<fqdn>,https://<fqdn>:9090,"Pulp, TFTP, DNS, DHCP, Puppet, Puppet CA, Logs, Dynflow, SSH, Discovery, Ansible, Openscap, Realm"

2018-06-23 05:53:31 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62147e80
2018-06-23 05:53:31 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62147eb8
2018-06-23 05:53:31 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:31 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  location info --name="Default Location"'
2018-06-23 05:53:34 - robottelo.ssh - INFO - <<< stdout
Id:                 2
Title:              Default Location
Name:               Default Location
Description:        
Users:              

Smart proxies:      
    <fqdn>
Subnets:            

Compute resources:  

Installation media: 
    CentOS mirror
    CoreOS mirror
    Debian mirror
    Fedora Atomic mirror
    Fedora mirror
    FreeBSD mirror
    mTe3Fk
    OpenSUSE mirror
    RancherOS mirror
    Ubuntu mirror
Templates:          
    Alterator default (provision)
    Alterator default finish (finish)
    Alterator default PXELinux (PXELinux)
    alterator_pkglist ()
    ansible_provisioning_callback ()
    ansible_tower_callback_script ()
    ansible_tower_callback_service ()
    Atomic Kickstart default (provision)
    AutoYaST default (provision)
    AutoYaST default iPXE (iPXE)
    AutoYaST default PXELinux (PXELinux)
    AutoYaST default user data (user_data)
    AutoYaST SLES default (provision)
    bmc_nic_setup ()
    Boot disk iPXE - generic host (Bootdisk)
    Boot disk iPXE - host (Bootdisk)
    chef_client ()
    coreos_cloudconfig ()
    CoreOS provision (provision)
    CoreOS PXELinux (PXELinux)
    create_users ()
    epel ()
    fix_hosts ()
    FreeBSD (mfsBSD) finish (finish)
    FreeBSD (mfsBSD) provision (provision)
    FreeBSD (mfsBSD) PXELinux (PXELinux)
    freeipa_register ()
    Grubby default (script)
    http_proxy ()
    Jumpstart default (provision)
    Jumpstart default finish (finish)
    Jumpstart default PXEGrub (PXEGrub)
    Junos default finish (finish)
    Junos default SLAX (provision)
    Junos default ZTP config (ZTP)
    Kickstart default finish (finish)
    Kickstart default iPXE (iPXE)
    Kickstart default PXEGrub (PXEGrub)
    Kickstart default PXEGrub2 (PXEGrub2)
    Kickstart default PXELinux (PXELinux)
    Kickstart default user data (user_data)
    kickstart_ifcfg_bonded_interface ()
    kickstart_ifcfg_bond_interface ()
    kickstart_ifcfg_generic_interface ()
    kickstart_ifcfg_get_identifier_names ()
    kickstart_networking_setup ()
    Kickstart oVirt-RHVH (provision)
    Kickstart oVirt-RHVH PXELinux (PXELinux)
    NX-OS default POAP setup (POAP)
    Preseed default (provision)
    Preseed default finish (finish)
    Preseed default iPXE (iPXE)
    Preseed default PXEGrub2 (PXEGrub2)
    Preseed default PXELinux (PXELinux)
    Preseed default user data (user_data)
    preseed_networking_setup ()
    puppet.conf ()
    puppetlabs_repo ()
    puppet_setup ()
    pxegrub2_chainload ()
    PXEGrub2 default local boot (PXEGrub2)
    pxegrub2_discovery ()
    PXEGrub2 global default (PXEGrub2)
    pxegrub_chainload ()
    PXEGrub default local boot (PXEGrub)
    pxegrub_discovery ()
    PXEGrub global default (PXEGrub)
    PXELinux chain iPXE (PXELinux)
    PXELinux chain iPXE UNDI (PXELinux)
    pxelinux_chainload ()
    PXELinux default local boot (PXELinux)
    PXELinux default memdisk (PXELinux)
    pxelinux_discovery ()
    PXELinux global default (PXELinux)
    rancheros_cloudconfig ()
    RancherOS provision (provision)
    RancherOS PXELinux (PXELinux)
    redhat_register ()
    remote_execution_ssh_keys ()
    saltstack_minion ()
    saltstack_setup ()
    UserData default (user_data)
    WAIK default PXELinux (PXELinux)
    XenServer default answerfile (provision)
    XenServer default finish (finish)
    XenServer default PXELinux (PXELinux)
Domains:            
    igc5nxb21k
Environments:       
    common
    development
    KT_6LZOQo_Library_jEGrOAJjgV_25
    KT_BtVvtEwkYrbM_Library_klSDVDE_252
    KT_gARliDj_Library_PbHFji_140
    KT_GRIVIinaCh_Library_omHFBzBhjJG_237
    KT_hbZEbNmn_Library_KLWCMdwQ_4
    KT_hbZEbNmn_Library_LVOSEYY_3
    KT_LrSnKoIeJLT_Library_SqXVHHcCMg_163
    KT_NpnlzPXNJaU_Library_GbXsaOy_100
    KT_psuJTS_Library_XwGsPKxCcu_334
    KT_rzmRc2_Library_PxxzHiAyHT_23
    KT_U7ZDja_Library_cccrZWrvAv_17
    KT_U7ZDja_Library_XgWEejTucn_18
    KT_ufpEaQ_Library_cqnCxahDULfP_343
    KT_VvWWLWO_Library_gRZgONet_196
    KT_WdNIks_Library_mprTXwLCstd_117
    KT_y9R5CI_Library_ESxdAFJUfH_386
    KT_yHtFxJnBzbI_Library_RsapmOYjKO_247
    KT_ZhyiwbFJj_Library_amdmkF_127
    KT_ZMsvENJ_Library_oFqBEC_225
    production
Hostgroups:         

Parameters:         

Created at:         2018/06/23 07:54:32
Updated at:         2018/06/23 07:54:32


2018-06-23 05:53:34 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62147eb8
2018-06-23 05:53:34 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62169b70
2018-06-23 05:53:34 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:34 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv domain create --location-ids="2" --name="o12lcbbval" --organizations="y9R5CI"'
2018-06-23 05:53:37 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Domain [o12lcbbval] created.,87,o12lcbbval

2018-06-23 05:53:37 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62169b70
2018-06-23 05:53:37 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b621704e0
2018-06-23 05:53:37 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:37 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  domain info --id="87"'
2018-06-23 05:53:39 - robottelo.ssh - INFO - <<< stdout
Id:            87
Name:          o12lcbbval
Description:   
DNS Id:        
Subnets:       

Locations:     
    Default Location
Organizations: 
    y9R5CI
Parameters:    

Created at:    2018/06/23 09:53:37
Updated at:    2018/06/23 09:53:37


2018-06-23 05:53:39 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b621704e0
2018-06-23 05:53:39 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62169668
2018-06-23 05:53:39 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:39 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  architecture info --name="x86_64"'
2018-06-23 05:53:42 - robottelo.ssh - INFO - <<< stdout
Id:                1
Name:              x86_64
Operating systems: 
    RedHat 7.5
Created at:        2018/06/23 07:54:32
Updated at:        2018/06/23 07:54:32


2018-06-23 05:53:42 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62169668
2018-06-23 05:53:42 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62169668
2018-06-23 05:53:42 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:42 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv os list --search="name="RedHat" AND major="6" OR major="7"" --per-page="10000"'
2018-06-23 05:53:44 - robottelo.ssh - INFO - <<< stdout
Id,Title,Release name,Family
1,RedHat 7.5,,Redhat

2018-06-23 05:53:44 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62169668
2018-06-23 05:53:44 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b626360f0
2018-06-23 05:53:44 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:44 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv partition-table list --operatingsystem-id="1" --per-page="10000"'
2018-06-23 05:53:46 - robottelo.ssh - INFO - <<< stdout
Id,Name,OS Family
94,Kickstart default,Redhat

2018-06-23 05:53:46 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b626360f0
2018-06-23 05:53:46 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62636780
2018-06-23 05:53:46 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:46 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv medium create --location-ids="2" --name="F3Jogw" --operatingsystem-ids="1" --organizations="y9R5CI" --path="http://OvCXDz"'
2018-06-23 05:53:49 - robottelo.ssh - INFO - <<< stdout
Message,Id,Name
Installation medium created.,109,F3Jogw

2018-06-23 05:53:49 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62636780
2018-06-23 05:53:49 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b62628400
2018-06-23 05:53:49 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:49 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme  medium info --id="109"'
2018-06-23 05:53:51 - robottelo.ssh - INFO - <<< stdout
Id:                109
Name:              F3Jogw
Path:              http://OvCXDz
OS Family:         
Operating systems: 
    RedHat 7.5
Locations:         
    Default Location
Organizations:     
    y9R5CI
Created at:        2018/06/23 09:53:48
Updated at:        2018/06/23 09:53:48


2018-06-23 05:53:51 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b62628400
2018-06-23 05:53:51 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f5b6202cf60
2018-06-23 05:53:51 - robottelo.ssh - INFO - Connected to [<fqdn>]
2018-06-23 05:53:51 - robottelo.ssh - INFO - >>> b'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv host create --architecture-id="1" --content-source-id="31792" --content-view-id="373" --domain-id="87" --ip="140.47.125.252" --lifecycle-environment-id="340" --location-id="2" --mac="9c:d6:b2:d2:80:a4" --medium-id="109" --name="srBKllKcOY" --operatingsystem-id="1" --organization="y9R5CI" --partition-table-id="94" --root-password="iqVclsjv"'
2018-06-23 05:53:54 - robottelo.ssh - INFO - <<< stderr
[ERROR 2018-06-23T05:53:54 API] 422 Unprocessable Entity
[ERROR 2018-06-23T05:53:54 Exception] Content source with given ID not found
Could not create the host:
  Content source with given ID not found

2018-06-23 05:53:54 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f5b6202cf60
2018-06-23 05:53:55 - robottelo - DEBUG - Finished Test: HostCreateTestCase/test_negative_create_with_content_source
```

that message is actually expected, we just fail to catch the proper exception:

```python
with self.assertRaises(CLIBaseError):
            make_fake_host({...})
```
and what we got:

```
E           robottelo.cli.factory.CLIFactoryError: Failed to create Host with data:{...}...
```

- the parent methods catch the excpetions and re-raise them finally as `CLIFactoryError` <- this is the one we should assert

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/cli/factory.py#L132

--- a/tests/foreman/cli/test_host.py
+++ b/tests/foreman/cli/test_host.py
@@ -380,7 +380,7 @@ class HostCreateTestCase(CLITestCase):
 
         :CaseImportance: Medium
         """
-        with self.assertRaises(CLIBaseError):
+        with self.assertRaises(CLIFactoryError):
             make_fake_host({
                 'content-source-id': gen_integer(10000, 99999),
                 'content-view-id': self.DEFAULT_CV['id'],

